tests/unit/test_dimension.py

```diff
@@ -1496,7 +1496,7 @@ def it_provides_access_to_the_addend_element_ids(
         assert subtotal.addend_ids == expected_value
 
     def it_knows_its_insertion_id_when_it_has_been_assigned_one(self):
-        assert _Subtotal({"insertion_id": 42}, None, None).insertion_id == 42
+        assert _Subtotal({"id": 42}, None, None).insertion_id == 42
```

Comment:
so this one stays. The two in `Describe_OrderSpec` go back to `"insertion_id":`.